client/src/components/Dashboard/Header/Authentication/Login.jsx

```diff
@@ -1,163 +1,105 @@
 import React, { Component } from 'react';
 import {
-  Form, Button, ButtonGroup, ToggleButton, Container, Row, Col, Alert,
+  Form, Button, Container, Row, Col, Alert,
 } from 'react-bootstrap';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
 import { Redirect } from 'react-router-dom';
-import { Toaster, Intent } from '@blueprintjs/core';
-import db, { facebookProvider, googleProvider } from '@src/fire';
+import * as loginActions from '../../../../actions/loginationActions';
```

Comment:
you can use alias like this:

import * as loginActions from '@src/actions/loginationActions';

Reply:
You can just use '@actions/loginationActions' here, it will work perfectly :)